spec/reports/cost_report_spec.rb

```diff
@@ -32,13 +38,40 @@
     end
   end
 
+  describe "making sure that access and rights come out the way they go in" do
+    it "pd == allow" do
+      cluster_tap_save [build(:ht_item, access: alo, rights: pd, ocns: [111])]
+      cluster = Cluster.find_by(ocns: 111)
+      expect(cluster.ht_items.count).to eq 1
+      expect(cluster.ht_items.first.rights).to eq pd
+      expect(cluster.ht_items.first.access).to eq alo
+    end
+    it "icus == allow" do
+      cluster_tap_save [build(:ht_item, access: alo, rights: icus, ocns: [222])]
```

Comment:
This would not happen with the current logic in the hathifiles; that said the behavior here is what we would expect in the holdings system

Reply:
Right, and the `ht_items` we get from `build(:ht_item, ...)` (the factorybot-defined /spec/factories/ht_item.rb) set rights and access independently from each other. If I just set `rights:icus` it's a coin toss whether that comes back `access:allow` or `access:deny`.